dcicutils/deployment_utils.py

```diff
@@ -708,7 +708,7 @@ def build_ini_stream_from_template(cls, template_file_name, init_file_stream, *,
         }
 
         # if we specify an indexer name for bs_env, we did the deployment wrong and should bail
-        if bs_env in INDEXER_ENVS:
+        if is_indexer_env(bs_env):
```

Comment:
@willronchetti, the function `is_indexer_env` will presently err so this code will fail. Would you rather I remove this clause or make `is_indexer_env` just unconditionally return `False`?`